blockchains/elastos/info/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Elastos",
+    "website": "https://elastos.info",
+    "description": "Elastos is an ecosystem of decentralized services supported by its own protocol blockchain with 50%+ of Bitcoin’s hashrate and multiple sidechains including a W3C Decentralized Identifier (DID) compliant identity chain and an industry standard Ethereum sidechain.",
```

Comment:
elastos folder should be testnet related, where would you put mainnet then? also without mainnet why you decided to add this testnet?